src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -210,7 +221,7 @@ class RequestorStep extends React.Component {
                                 dob: this.state.dob,
                                 ssnLast4: this.state.ssnLast4,
                             }}
-                            errors={this.props.reimbursementAccount.errors}
+                            errors={this.props.reimbursementAccount.errorFields}
```

Comment:
Are we returning `errorFields` in the API? 🤔

Reply:
This was renamed [here](https://github.com/Expensify/App/pull/11162/files#diff-636e588f5050947f85a82e4c67a5bf4bf984b5df3b7ff71fd9afb8e7118a977aL6), to avoid confusion between errors and errorFields

**errors** are global errors from the api and will be displayed in reimbursementAccountForm.js
**errorFields** is set JS side and is the map of fieldName to bool, indicating which field to highlight as error